pkg/storage/postgresql/targets.go

```diff
@@ -242,6 +248,8 @@ func (db DB) TargetsSummary(filter storage.Filter) ([]model.TargetSummary, stora
 				{{ end }}
 			{{ end }}
 			{{ if and (eq .Status "OPEN") (or .MinDate .MaxDate) }}
+				/* Ignore findings marked as false positive */
+				AND f.status != 'FALSE_POSITIVE'
 				{{ if .MinDate }}
 				AND fexp.found_at >= :minDate
 				{{ end }}
```

Comment:
Same as `ListFindings` query, I think we are missing the ignore FALSE POSITIVE findings when the `atDate` filter is used without a `status` filter?

Reply:
If you don't mind, I will resolve this comment when we agree on what to do with the ListFindings method.